### DISCUSSION TITLE
Create an automatic order number on persist

### DISCUSSION THREAD

**Discussion by anleclerc** [symfony/symfony#53961] [Q&A]
Hi,

I'd like to know if it's possible to create automatic order number on persist.

In ma table I have an primary key which is an uuid, and I'd like to have an order number (ex: W000001) which is automatically created when I persist entity. I'don't want order number as primary key but really as specific column.

Any help ?

**Comment by MatTheCat** [comment]
Hello,

look at the [StofDoctrineExtensionsBundle](https://symfony.com/bundles/StofDoctrineExtensionsBundle/current/index.html), it provides [a sortable extension](https://github.com/doctrine-extensions/DoctrineExtensions/blob/main/doc/sortable.md).
